Classes/LanguageDetection.php

```diff
@@ -452,7 +475,7 @@ protected function getSysLanguages() {
 			}
 		} else {
 			$res = $this->getDB()->exec_SELECTquery(
-				'sys_language.uid, sys_language.title as isocode',
```

Comment:
In TYPO3 7 at least inside "sys_language.language_isocode" we have something like "en" whereas there is something like ""English" inside the fields "sys_language.title". The Results go in the array "$availableLanguages".

Later on in Row 474 ff we compare the ISO-CODES in $availableLanguages against the col "lg_iso_2".

In Line 166ff we fetch the result of getAcceptedLanguages() and compare against the list of browser-accepted languages in the form "en,fr,de" or something like "US-en,FR-fr,DE-de".

Without that change I don't get any accepted languages since "english" is not in the list of browser-accepted languages. Maybe i misunderstood something here?

Is there any reason to fetch the title instead of the iso-code?